packages/core/src/exposed-thing.ts

```diff
@@ -635,331 +465,6 @@ export default class ExposedThing extends TD.Thing implements WoT.ExposedThing {
-                // let next = listener;
-                // let error = null;
-                // let complete = null;
-                // let sub: Subject<Content> = this.properties[name].getState().subject;
-                // sub.asObservable().subscribe(next, error, complete);
-                ps.listeners.push(listener);
-                console.debug("[core/exposed-thing]", `ExposedThing '${this.title}' subscribes to property '${name}'`);
-            } else {
-                reject(new Error(`ExposedThing '${this.title}', no property found for '${name}'`));
-            }
-        });
-    }
-
-    /**
-     * @deprecated
-     */
-    public unobserveProperty(name: string): Promise<void> {
-        return new Promise<void>((resolve, reject) => {
-            if (this.properties[name]) {
-                // let sub: Subject<Content> = this.properties[name].getState().subject;
-                // sub.unsubscribe();  // XXX causes loop issue (see browser counter example)
-                console.debug(
-                    "[core/exposed-thing]",
-                    `ExposedThing '${this.title}' unsubscribes from property '${name}'`
-                );
-            } else {
-                reject(new Error(`ExposedThing '${this.title}', no property found for '${name}'`));
-            }
-        });
-    }
-
-    /**
-     * @deprecated
-     */
-    public subscribeEvent(name: string, listener: WoT.WotListener, options?: WoT.InteractionOptions): Promise<void> {
-        return new Promise<void>((resolve, reject) => {
-            if (this.events[name]) {
-                const es: EventState = this.events[name].getState();
-                // let next = listener;
-                // let error = null;
-                // let complete = null;
-                // let sub: Subject<any> = this.events[name].getState().subject;
-                // sub.asObservable().subscribe(next, error, complete);
-                es.legacyListeners.push(listener);
-                // es.subject.asObservable().subscribe(listener, null, null);
-                console.debug("[core/exposed-thing]", `ExposedThing '${this.title}' subscribes to event '${name}'`);
-            } else {
-                reject(new Error(`ExposedThing '${this.title}', no event found for '${name}'`));
-            }
-        });
-    }
-
-    /**
-     * @deprecated
-     */
-    public unsubscribeEvent(name: string): Promise<void> {
-        return new Promise<void>((resolve, reject) => {
-            if (this.events[name]) {
-                // let sub: Subject<any> = this.events[name].getState().subject;
-                // sub.unsubscribe(); // XXX causes loop issue (see browser counter example)
-                console.debug("[core/exposed-thing]", `ExposedThing '${this.title}' unsubscribes from event '${name}'`);
-            } else {
-                reject(new Error(`ExposedThing '${this.title}', no event found for '${name}'`));
-            }
-        });
-    }
-
```

Comment:
These methods were all public. In not against removing them, but in practice this change causes a public API change which means that the next version will be a 0.9.x if we care about semver.

Reply:
Mhh, good point... internally they were net referenced anymore... and incrementing to 0.9. seems like an overkill to me 🙃

Reply:
yeah I know, but it is just a number 😄 I think we should start issuing releases more often. Take for example the caop library. If they had some release process like us we would still have a lot of bugs.

Reply:
I see your point. Let's discuss this in https://github.com/eclipse/thingweb.node-wot/pull/727#issuecomment-1092634350